modules/terraform/Makefile

```diff
@@ -34,6 +34,15 @@ terraform/validate:
 terraform/tflint:
 	tflint --enable-plugin=aws
 
+## Auto-fix terraform deprecated list accessors
+terraform/tflint-fix-deprecated-index:
+	@for v in $$(find . -type f -name '*.tf'); do \
+		sed -i 's/\([[:alpha:]]\+[0-9]*\)\.\([*0-9]\)\./\1[\2]\./g' "$$v" ; \
```

Comment:
I'm not happy with this regex. It would convert `v1.2.3` to `v1[2].3`. Let's use this to require at least 3 characters for the identifier part. I would rather miss some than change some incorrectly.

```suggestion
		sed -i 's/\([[:alpha:]][-_[:alpha:][:digit:]]\{2,\}\)\.\([*0-9]\+\)\./\1[\2]\./g' "$$v" ; \
```

Reply:
`ec2`, `s3` - won't work 😄  Tried that